daily_jobs.py

```diff
@@ -0,0 +1,202 @@
+{%- endfor %}{% endif %}
+
+To view the abstracts and register please visit the [Speakers' Corner page](https://virtualscienceforum.org/#/speakers-corner).
+
+Best regards,
+The VSF Organizers
+
+---
+You are receiving this email because you are signed up for the Speakers' Corner weekly program updates.
+To unsubscribe visit [this URL](%mailing_list_unsubscribe_url%).
+""")
+
+def email_video_link(talk):
+    """Send the presenter a link to their video, asking to confirm."""
+    meeting_recordings = common.zoom_request(
+        requests.get,
+        common.ZOOM_API + f"/meetings/{talk['zoom_meeting_id']}/recordings"
+    )
+    if not len(meeting_recordings["recording_files"]):
+        raise RuntimeError("No recordings found")
+
+    message = RECORDING_AVAILABLE_TEMPLATE.render(
+        share_url=meeting_recordings["share_url"],
+        **talk,
+    )
+
+    response = common.api_query(
+        requests.post,
+        common.MAILGUN_DOMAIN + "messages",
+        data={
+            "from": "VSF team <no-reply@mail.virtualscienceforum.org>",
+            "to": f"{talk['speaker_name']} <{talk['email']}>",
+            "subject": "Approve your Speakers' Corner recording",
+            "text": common.markdown_to_plain(message),
+            "html": common.markdown_to_email(message),
+        }
+    )
+    logging.info(f"Notified the speaker of {talk['zoom_meeting_id']} about recording.")
+    return response
+
+def weekly_speakers_corner_update(talks):
+    # Filter out only the speakers' corner talks
+    now = datetime.datetime.now(tz=pytz.UTC)
+    week = datetime.timedelta(days=7)
+    this_week_talks = list(filter(
+        (lambda talk: now < talk["time"] < now + week),
+        talks
+    ))
+    next_week_talks = list(filter(
+        (lambda talk: now + week < talk["time"] < now + 2*week),
+        talks
+    ))
+
+    if not any([this_week_talks, next_week_talks]):
+        # Nothing to announce
+        return
+
+    message = WEEKLY_ANNOUNCEMENT_TEMPLATE.render(
+        this_week_talks=this_week_talks,
+        next_week_talks=next_week_talks,
+    )
+    data = {
+        "from": "VSF team <no-reply@mail.virtualscienceforum.org>",
+        "to": "speakers_corner@mail.virtualscienceforum.org",
+        "subject": "Speakers' Corner weekly schedule",
+        "text": common.markdown_to_plain(message),
+        "html": common.markdown_to_email(message),
+    }
+
+    response = common.api_query(
+        requests.post,
+        common.MAILGUN_DOMAIN + "messages",
+        data=data
+    )
+    logging.info("Sent the weekly update.")
+    return response
+
+
+def add_zoom_registrants_to_mailgun():
+    """Add registrants from Zoom to Mailgun mailing list."""
+    # Get all meetings
+    meetings = common.all_meetings("me")
+
+    # Filter by past meetings
+    past_lrc = [
+        meeting for meeting in meetings
+        if ("Long Range" in meeting['topic']
```

Comment:
We also have workshops, and those have a lot of sign-ups. Let's rather filter by checking whether registration is enabled.

Reply:
The API for getting a user's meetings unfortunately does not return a meeting's settings (which has the registration type). We could just literally use all meetings instead for now?

Reply:
Yes, that sounds like a good approach, as long as we take care to catch the errors that might arise from meetings without registrations.

Reply:
I've updated common.py's registrants() to do so. A successful request for registrants does not return a success code somehow, only a failed one does (code 300). So the current check is rather ugly, suggestions?

Reply:
Fixed. `response.json()` isn't a response and doesn't have a status code :)